packages/shared/lib/core/nfts/stores/selected-account-nfts.store.ts

```diff
@@ -15,3 +15,18 @@ export const selectedAccountNfts: Readable<INft[]> = derived(
         }
     }
 )
+
+export const nftSearchTerm: Writable<string> = writable('')
+
+export const queriedNfts: Readable<INft[]> = derived(
+    [selectedAccountNfts, nftSearchTerm],
+    ([$selectedAccountNfts, $nftSearchTerm]) => {
+        let nftList = $selectedAccountNfts
+
+        if ($nftSearchTerm) {
+            nftList = nftList.filter((nft) => nft.name.toLowerCase().includes($nftSearchTerm.toLowerCase()))
+        }
+
+        return nftList.sort((nft1, nft2) => (nft2.name < nft1.name ? 1 : -1))
```

Comment:
This doesn't sort alphabetically; it's sorted by unicode value. "NFT" is placed before "application". It's not an issue but we should figure out how we want to default sort the NFTs.

Reply:
ah thats true, i'll change this to `nft2.name.toLowercase() < nft1.name.toLowercase()`